ui-mockups/src/articles.html

```diff
@@ -87,20 +119,24 @@ <h6><a href="article.html">Donec id elit non mi porta</a></h6>
                                 </small>
                             </p>
                             <small>
-                                <a class="ah-author-link" href="#">Meshack Mbuvi</a>
+                                By <a class="ah-author-link" href="#">Mbuvi</a>
                             </small>
-                            <p class="d-flex justify-content-between">
+                            <p class="d-flex justify-content-between article-footer">
```

Comment:
@meshack-mbuvi Is the lower border necessary. Also I think the colors do not really blend into the existing template we have.

Reply:
Which color do you recommend for the likes and dislikes @PAbishai ?

Reply:
Maybe space the icons and the number of likes a bit and tint the color a bit to make it lighter and blend in more with the other colors